examples/home_tab.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+import Slack as Slack
+
+from slackify import Slackify, Flask, Slack, reply_text
+
+# Initialize App
+app = Flask(__name__)
+slackify = Slackify(app = app)
+cli = Slack(os.getenv('SLACK_BOT_TOKEN'))
+
+import os
+from flask import Flask
```

Comment:
Please move all imports at the top level

Reply:
Will do.